metamorphic/options.go

```diff
@@ -339,6 +362,11 @@ func standardOptions() []*TestOptions {
 		25: `
 [TestOptions]
   enable_value_blocks=true
+`,
+		26: `
+[TestOptions]
```

Comment:
The standard option looks great! maybe we should also add an ability for the random options below to toggle these?

Reply:
Will do.